nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_continental/continental_srr520_hw_interface.hpp

```diff
@@ -0,0 +1,151 @@
+#include <nebula_msgs/msg/nebula_packets.hpp>
+
+#include <memory>
+#include <mutex>
+#include <string>
+#include <vector>
+
+namespace nebula
+{
+namespace drivers
+{
+namespace continental_srr520
+{
+/// @brief Hardware interface of the Continental SRR520 radar
+class ContinentalSRR520HwInterface
+{
+public:
+  /// @brief Constructor
+  ContinentalSRR520HwInterface();
+
+  /// @brief Starting the interface that handles UDP streams
+  /// @return Resulting status
+  Status SensorInterfaceStart();
+
+  /// @brief Function for stopping the interface that handles UDP streams
+  /// @return Resulting status
+  Status SensorInterfaceStop();
+
+  /// @brief Setting sensor configuration
+  /// @param sensor_configuration SensorConfiguration for this interface
+  /// @return Resulting status
+  Status SetSensorConfiguration(
+    const std::shared_ptr<
+      const nebula::drivers::continental_srr520::ContinentalSRR520SensorConfiguration>
+      new_config_ptr);
+
+  /// @brief Registering callback for PandarScan
+  /// @param scan_callback Callback function
+  /// @return Resulting status
+  Status RegisterPacketCallback(
+    std::function<void(std::unique_ptr<nebula_msgs::msg::NebulaPacket>)> packet_callback);
+
+  /// @brief Sensor synchronization routine
+  void SensorSync();
+
+  /// @brief Process a new Sync Fup request
+  void SensorSyncFup(builtin_interfaces::msg::Time stamp);
+
+  /// @brief Configure the sensor
+  /// @param sensor_id Desired sensor id
+  /// @param longitudinal_autosar Desired longitudinal value in autosar coordinates
+  /// @param lateral_autosar Desired lateral value in autosar coordinates
+  /// @param vertical_autosar Desired vertical value in autosar coordinates
+  /// @param yaw_autosar Desired yaw value in autosar coordinates
+  /// @param longitudinal_cog Desired longitudinal cog
+  /// @param wheelbase Desired wheelbase
+  /// @param cover_damping Desired cover damping
+  /// @param plug_bottom Desired plug bottom
+  /// @param reset Rest the sensor to its default values
+  /// @return Resulting status
+  Status ConfigureSensor(
+    uint8_t sensor_id, float longitudinal_autosar, float lateral_autosar, float vertical_autosar,
+    float yaw_autosar, float longitudinal_cog, float wheelbase, float cover_damping,
+    bool plug_bottom, bool reset);
+
+  /// @brief Set the current vehicle dynamics
+  /// @param longitudinal_acceleration Longitudinal acceleration
+  /// @param lateral_acceleration Lateral acceleration
+  /// @param yaw_rate Yaw rate
+  /// @param longitudinal_velocity Longitudinal velocity
+  /// @param driving_direction Driving direction
+  /// @return Resulting status
+  Status SetVehicleDynamics(
+    float longitudinal_acceleration, float lateral_acceleration, float yaw_rate,
+    float longitudinal_velocity, bool standstill);
+
+  /// @brief Checking the current settings and changing the difference point
+  /// @return Resulting status
+  Status CheckAndSetConfig();
+
```

Comment:
This is not used and not defined, please delete.
```suggestion
```

Reply:
Addressed in [c3e6ea1](https://github.com/tier4/nebula/pull/124/commits/c3e6ea1c2fc761b9e14fd77c20a99e12553e81fb)
(the same was applied to the ARS548)
:bow: